cmd/system-shutdown/system-shutdown.c

```diff
@@ -93,7 +101,19 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	reboot(cmd);
+	// glibc reboot wrapper does not expose the optional reboot syscall
+	// parameter
+
+	long ret;
+	if (cmd == RB_AUTOBOOT && reboot_arg[0] != '\0')
+		ret = syscall(SYS_reboot,
+			      LINUX_REBOOT_MAGIC1, LINUX_REBOOT_MAGIC2,
+			      LINUX_REBOOT_CMD_RESTART2, reboot_arg);
+	else
+		ret = reboot(cmd);
+
+	if (ret == -1)
+		kmsg("Error calling reboot! : %s (%d)", strerror(errno), errno);
```

Comment:
Nitpick, can you please reword this: `kmsg(cannot reboot the system: %s", strerror(errno));`